CodeEdit/Features/LSP/LSPUtil.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  LSPUtil.swift
+//  CodeEdit
+//
+//  Created by Abe Malla on 2/10/24.
+//
+
+import Foundation
+import LanguageServerProtocol
+
+/// Helper function to get the edits from a completion item
+/// - Parameters:
+///  - startPosition: The position where the completion was requested
+///  - item: The completion item
+///  - Returns: An array of TextEdit objects
+func getCompletionItemEdits(startPosition: Position, item: CompletionItem) -> [TextEdit] {
```

Comment:
This is a generic, project wide, function. Is that on purpose? Such loose functions may not be a good idea, until we maybe introduce modules again

The function also is quite big and has a lot of logic branches

Reply:
Done